config/routes.rb

```diff
@@ -18,7 +18,9 @@
     get 'profile', to: 'users#show'
 
     resource :education_benefits_claims, only: :create
-    resources :claims, only: [:index]
+    resources :claims, only: [:index] do
+      post :documents, on: :member
```

Comment:
I would make documents a subresource of claims even if it's only the `create` action right now